js/okx.js

```diff
@@ -1868,13 +1868,21 @@ module.exports = class okx extends Exchange {
         const takeProfitTrigger = this.safeNumber (params, 'tpTriggerPx');
         params = this.omit (params, [ 'slOrdPx', 'stopPrice', 'tpOrdPx', 'tpTriggerPx', 'slTriggerPx' ]);
         if (stopPrice || stopTrigger) {
-            defaultMethod = 'privatePostTradeOrderAlgo';
-            request['slOrdPx'] = this.priceToPrecision (symbol, stopPrice);
-            request['slTriggerPx'] = this.priceToPrecision (symbol, stopTrigger);
+            if (stopPrice && stopTrigger) {
+                defaultMethod = 'privatePostTradeOrderAlgo';
+                request['slOrdPx'] = this.priceToPrecision (symbol, stopPrice);
```

Comment:
We should not use `stopPrice` for stop-loss orders, the `stopPrice` is used for stop-orders only:

- Stop orders use `stopPrice` << we should only unify this at this time.
- Stop-loss orders use `stopLossPrice` <<  this should not be mixed or confused with stopPrice
- Take-profit orders use `takeProfitPrice` << this should not be mixed or confused with stopPrice